other/materials_designer/create_interface_with_relaxation_ase_emt.ipynb

```diff
@@ -4,436 +4,205 @@
    "cell_type": "markdown",
```

Comment:
<p>Typo in first sentence. Interface.</p><p> </p><p>EMT potential in the title</p><p> </p><p>Notes are irrelevant to this notebook</p><p> </p><p>You have to explain that one can first run the ZSL notebook and then this one above to do the relaxation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>